### PR TITLE
Button to create a new playlist

### PR DESCRIPTION
**Pull request by errietta**

### CODE REVIEW

**Comment by try-server-hook** [comment]
[errietta Errietta Kostala (errietta)](https://github.com/errietta) started tests. [Results](https://treeherder.mozilla.org/ui/#/jobs?repo=gaia-try&revision=19e48c0924be)

File: build/config/common-settings.json [hunk 1]
```diff
@@ -59,6 +59,7 @@
    "devtools.eventlooplag.threshold": 100,
    "devtools.remote.wifi.visible": false,
    "devtools.remote.wifi.enabled": false,
+   "devtools.debugger.remote-enabled": true,
    "feedback.url": "https://input.allizom.org/api/v1/feedback/",
```
Comment (hfiguiere) [inline build/config/common-settings.json]: I'm pretty sure you didn't mean to commit any of these changes.

Comment (errietta) [inline build/config/common-settings.json]: oops :D

**Comment by try-server-hook** [comment]
[errietta Errietta Kostala (errietta)](https://github.com/errietta) started tests. [Results](https://treeherder.mozilla.org/ui/#/jobs?repo=gaia-try&revision=8766ddb0c429)

**Comment by errietta** [comment]
let's keep one PR for these chages, https://github.com/mozilla-b2g/gaia/pull/27125

**Comment by hfiguiere** [comment]
:+1:

**Pull request closed without merging**